src/material-experimental/mdc-checkbox/checkbox.ts

```diff
@@ -210,28 +259,7 @@ export class MatCheckbox implements AfterViewInit, OnDestroy, ControlValueAccess
   private _attrBlacklist = new Set(['aria-checked']);
 
   /** The `MDCCheckboxAdapter` instance for this checkbox. */
-  private _checkboxAdapter: MDCCheckboxAdapter = {
-    addClass: (className) => this._setClass(className, true),
-    removeClass: (className) => this._setClass(className, false),
-    forceLayout: () => this._checkbox.nativeElement.offsetWidth,
-    hasNativeControl: () => !!this._nativeCheckbox,
-    isAttachedToDOM: () => !!this._checkbox.nativeElement.parentNode,
-    isChecked: () => this.checked,
-    isIndeterminate: () => this.indeterminate,
-    removeNativeControlAttr:
-        (attr) => {
-          if (!this._attrBlacklist.has(attr)) {
-            this._nativeCheckbox.nativeElement.removeAttribute(attr);
-          }
-        },
-    setNativeControlAttr:
-        (attr, value) => {
-          if (!this._attrBlacklist.has(attr)) {
-            this._nativeCheckbox.nativeElement.setAttribute(attr, value);
-          }
-        },
-    setNativeControlDisabled: (disabled) => this.disabled = disabled,
-  };
+  private _adapter: CheckBoxAdapter;
```

Comment:
You probably don't need this anymore (can just call new CheckBoxAdaptor() when creating the foundation